storage/innobase/handler/ha_innodb.cc

```diff
@@ -15458,8 +15458,8 @@ delete is then allowed internally to resolve a duplicate key conflict in
 REPLACE, not an update.
 @return > 0 if referenced by a FOREIGN KEY */
 
-uint
-ha_innobase::referenced_by_foreign_key(void)
+bool
+ha_innobase::referenced_by_foreign_key() const noexcept
 /*========================================*/
```

Comment:
The `@return` comment needs to be adjusted.

While you are touching this, you should also remove the decoration or uglification comment below the function. Heikki Tuuri wanted to have such comments, as well as an empty line before the return type, which he wanted to have in a line on its own. In the current formatting style, this should be as follows:
```c++
/** …
@return whether the table is referenced by a FOREIGN KEY */
bool ha_innobase::referenced_by_foreign_key() const noexcept
```

Reply:
Again, I think such a coding style change is unrelated and should be a separate commit - but I changed it.